charts/cortex/templates/ingester/ingester-svc.yaml

```diff
@@ -15,4 +16,38 @@ spec:
       protocol: TCP
       name: http-metrics
       targetPort: http-metrics
+    - port: {{ .Values.config.server.grpc_listen_port }}
+      protocol: TCP
+      name: grpc
+      targetPort: grpc
   selector: {{- include "cortex-ingester.matchLabels" . | nindent 4 }}
+{{- end }}
+{{- if .Values.topologyAware.enabled }}
+{{- range $availabilityCell := .Values.topologyAware.availabilityCells }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "cortex-ingester.fullname" $ }}-{{ $availabilityCell }}
+  labels: {{ include "cortex-ingester.labels" $ | nindent 4 }}
+    {{- with $.Values.ingester.service.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    {{ $.Values.topologyAware.zoneLabel }}: {{ $availabilityCell }}
+  annotations:
+    {{- toYaml $.Values.ingester.service.annotations | nindent 4 }}
+spec:
+  type: ClusterIP
```

Comment:
I think the service used for the statefulset should normally be headless, i.e., it needs `clusterIP: None` which is shy it cannot be used for "normal" loadbalancing of the incoming trafic.